radio/src/opentx.cpp

```diff
@@ -1728,9 +1728,15 @@ void copyTrimsToOffset(uint8_t ch)
   storageDirty(EE_MODEL);
 }
 
+
 #if defined(STARTUP_ANIMATION)
-#define PWR_PRESS_DURATION_MIN        100 // 1s
-#define PWR_PRESS_DURATION_MAX        500 // 5s
+
+inline uint32_t PWR_PRESS_DURATION_MIN()
```

Comment:
perhaps uint8_t ?

Reply:
it seems the unit is not the same than PWR_PRESS_DURATION_MIN, it's prone to mistakes!